pkg/build/nodeimage/const_cni.go

```diff
@@ -105,6 +105,8 @@ spec:
               fieldPath: status.podIP
         - name: POD_SUBNET
           value: {{ .PodSubnet }}
+        - name: CONTROL_PLANE_ENDPOINT
+          value: {{ .ControlPlaneEndpoint }}
```

Comment:
Note: this is backwards incompatible.

Reply:
in which sense, the ci is passing with this change ... (I´m a bit slow today)

Reply:
older kind won't be able to run this image

Reply:
:thinking: what will go template do?

Reply:
Explode on indexing a field that doesn't exist